src/settings.js

```diff
@@ -1799,6 +1799,11 @@ var WASM2C = 0;
 // in a custom location.
 var SEPARATE_DWARF_URL = '';
 
+// Add a #! line output JavaScript file and make it executable.  This command
+// used to run the output can be configured via the .emscripten config file using
+// the JS_ENGINE setting and defaults to NODE_JS.
+var EXECUTABLE = '';
```

Comment:
how about `EXECUTABLE_JS`? or do you think it'll be useful for non-JS too in standalone mode or elsewhere?

Reply:
I don't really care what this is called.  If you prefer that name I will change it.

I could even make this an internal setting in this change?  But it makes it less testable I guess?

Reply:
I'm ok with `EXECUTABLE`, just wanted to raise some questions.